src/lib/widgets/episode/ui/episode-card.svelte

```diff
@@ -1,17 +1,30 @@
 <script lang="ts">
+  import type { Episode } from '$lib/shared/api';
   import { PlaybackButton } from '$lib/features/playback-controls';
   import { LikeButton } from '$lib/features/like-episode';
   import { DownloadLink } from '$lib/features/download-episode';
   import { EpisodeCardShell } from '$lib/entities/episode';
-  import type { Episode } from '$lib/shared/api';
+  import { play, pause, audio } from '$lib/entities/audio';
+  import { addToListeningHistory } from '$lib/entities/audio/model/listening-history';
 
   export let episode: Episode;
-  export let playing: boolean;
+  export let podcastId: number | null = null;
+
+  $: playing = $audio?.src === episode.audioUrl && !$audio?.paused && !$audio?.ended;
+
+  const playAudio = () => {
```

Comment:
issue: I'm concerned that this duplicates the logic of `features/playback-controls/model/toggle-global-playback`

suggestion: could we edit that one instead and use it here?